src/gpuhunt/_internal/models.py

```diff
@@ -33,6 +33,7 @@ class AcceleratorVendor(str, enum.Enum):
     AMD = "amd"
     GOOGLE = "google"
     INTEL = "intel"
+    TENSTORRENT = "tenstorrent"
```

Comment:
I suggest using "Tenstorrent" in other names as well (for consistency):

`KNOWN_TENSTORRENT_ACCELERATORS`, `TenstorrentAcceleratorInfo`